nx.json

```diff
@@ -0,0 +1,54 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "lint",
+          "build",
+          "test",
+          "test:contracts",
+          "prettier",
+          "solhint",
+          "test:forge",
+          "test:hardhat",
+          "test:solidity",
+          "test:typescript",
+          "build:solidity",
+          "build:typescript"
+        ]
+      }
+    }
+  },
+  "targetDefaults": {
+    "dev": {
+      "dependsOn": ["^build"]
```

Comment:
The `^build` means it depends on dependencies builds finishing before it can run
`build` without the ^ would mean it depends on THIS packages builds not dependencies